src/components/Facility/FacilityHome.tsx

```diff
@@ -191,12 +191,12 @@ export const FacilityHome = ({ facilityId }: Props) => {
                 <div className="relative rounded-3xl  h-full w-full bg-[radial-gradient(circle_at_50%_120%,rgba(255,255,255,0.2),transparent)]" />
               )}
               <div className="absolute bottom-0 left-0 translate-x-0 translate-y-1/3">
-                <Button variant="link">
+                <div className="px-4 inline-flex rounded-md [&_svg]:size-4">
```

Comment:
why?

```suggestion
                <div className="px-4 inline-flex rounded-md">
```

Reply:
> this property: `[&_svg]:size-4` is used to make sure the svg inside the avatar is of size-4

idea is to solve anything that feels wrong... even if it means it was wrong previously 😅